pages/common/asar.md

```diff
@@ -13,7 +13,7 @@
 
 - Extract a specific file from an archive:
 
-`asar extract-file {{archived.asar}} {{file}}`
+`asar extract-file {{archived.asar}} {{path/to/file}}`
```

Comment:
```suggestion
`asar extract-file {{path/to/archive.asar}} {{path/to/file}}`
```
Can you do the same for other examples in this file?

Reply:
Sure. :) A little bit later.